src/main/java/com/mraof/minestuck/item/crafting/alchemy/GristTypes.java

```diff
@@ -41,6 +41,7 @@
 	public static final RegistryObject<GristType> ARTIFACT = GRIST_TYPES.register("artifact", () -> new GristType(new GristType.Properties(0.1F, 1).candy(() -> MSItems.ARTIFACT_WARHEAD)));
 	public static final RegistryObject<GristType> ZILLIUM = GRIST_TYPES.register("zillium", () -> new GristType(new GristType.Properties(0.0F, 10).candy(() -> MSItems.ZILLIUM_SKITTLES).notUnderlingType()));
 	
+	
```

Comment:
Try to avoid accidental line creation/removal like this. I typically check through everything I changed in the Commit changelist before going through with the commit to make sure I didnt do so. Its also good practice to make sure there arent any unused imports at the same time. In order to ensure I am adding changes that I wanted, and to document when I made the changes, I add a relevant Commit Message(also in the Commit window with the changelist)